src/script-child-process.ts

```diff
@@ -35,18 +38,25 @@ const PATH_ENV_SUFFIX = (() => {
   return entries.slice(endOfNodeModuleBins).join(pathlib.delimiter);
 })();
 
+export type ScriptChildProcessState =
+  | 'starting'
+  | 'started'
+  | 'stopping'
```

Comment:
'stopping' could be a bit clearer, it means that we're trying to kill the process yeah?

Reply:
Done. Renamed to `killing`.